local-trial

```diff
@@ -176,10 +176,8 @@ if retool_trial_running; then
   $MAYBE_SUDO docker-compose down
 fi
 
-if [ ! -f ./docker-compose-ssl.yml ]; then
+if [ ! -f ./docker-compose-ssl.yml ] && [ -f ./docker-compose-local-trial.yml ]; then
```

Comment:
This is more backward compatible (ran into some issues with someone not having this file).